### DISCUSSION TITLE
How can I detect scrolling within a Select Content? (For pagination)

### DISCUSSION THREAD

**Discussion by StephenSHorton** [radix-ui/primitives#2758] [Help] [answered]
I'd like to paginate more select items in after the user has scrolled a certain amount in the select content container. But it appears that `onScroll` doesn't fire at all when scrolling.

**Comment by benoitgrelard** [comment, marked as answer]
Maybe you're attaching the handler to the wrong part? It should be on the `Viewport` part, see video below:

https://github.com/radix-ui/primitives/assets/1539897/4f2609d5-35fc-4116-9670-69e0cfaafcc3

**Reply by StephenSHorton** [reply]
You are the best @benoitgrelard!
![image](https://github.com/radix-ui/primitives/assets/54323940/af21a24e-aa4b-4cc9-968b-e344486f458d)
